src/hotspot/share/services/stathist.cpp

```diff
@@ -1132,7 +1132,7 @@ void cleanup() {
 
 void print_report(outputStream* st, const print_info_t* pi) {
 
-  st->print("stathist:");
+  st->print("Vitals History:");
```

Comment:
I would omit the word "History".  As I understand, there is no printout of a single sample which would justify to distinguish "Vitals" and "Vitals History"?